sslcontext-kickstart-for-pem/src/main/java/nl/altindag/ssl/util/PemFormatter.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2019 Thunderberry.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.altindag.ssl.util;
+
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+final class PemFormatter {
+
+    private static final Pattern PEM_PATTERN = Pattern.compile("(-----.*?-----)(.*?)(-----.*?-----)");
```

Comment:
What you could add is possible BEGIN and END patterns but this is also fine :)

Reply:
Thank you for the suggestion, I applied it!